packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -170,6 +173,32 @@ exports.list = async (req, res, journeyId) => {
 	}); //todo: use layout property on HASJourney object
 };
 
+/**
+ * builds a question page view model
+ * @param {Journey} journey
+ * @param {Section} section
+ * @param {Question} question
+ * @param {Answer} answer
+ * @returns {Object}
+ */
+function getQuestionViewModel(journey, section, question, answer) {
+	const backLink = journey.getNextQuestionUrl(section.segment, question.fieldName, true);
```

Comment:
this was pointing at current url in validation scenarios, any reason as to why, or this ok?

Reply:
Yeah I'm not sure as to thy the validation would be affecting it in this way. If it wasn't working as expected and this does then it's all good with me.